pkg/autoscaler/autoscaler_test.go

```diff
@@ -307,29 +307,6 @@ func TestAutoscaler_PanicThenUnPanic_ScaleDown(t *testing.T) {
 	a.expectScale(t, now, 10, true) // back to stable mode
 }
 
-func TestAutoscaler_PodsAreWeightedBasedOnLatestStatTime(t *testing.T) {
-	a := newTestAutoscaler(10.0)
-	now := a.recordLinearSeries(
-		t,
-		roundedNow(),
-		linearSeries{
-			startConcurrency: 10,
-			endConcurrency:   10,
-			duration:         30 * time.Second,
-			podCount:         10,
-		})
-	now = a.recordLinearSeries(
-		t,
-		roundedNow(),
-		linearSeries{
-			startConcurrency: 0,
-			endConcurrency:   0,
-			duration:         30 * time.Second,
-			podCount:         10,
-		})
-	a.expectScale(t, now, 5, true) // 10 pods lameducked half the time count for 5
-}
-
```

Comment:
Is this diff necessary? The test seems valid?

Reply:
The test is valid but what it tested now is covered by other tests. It used to test how we calculate pod weight. Now we drop the pod weight so it's no longer needed.